__tests__/app.test.js

```diff
@@ -174,3 +174,67 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST: /api/articles/:article_id/comments", () => {
```

Comment:
Good tests, would suggest a test for invalid id, ie a string.
Also a test for the post request to succeed if extra properties are provided on the request body, and are ignored.

Reply:
“a test for invalid id” - this scenario is already covered in GET / articles / article_id (via https://github.com/Anna702/NC-NEWS-project/blob/071c2f112fb3ecea6320b61087a8a983cf656578/controllers/errorHandlingControllers.js#L8 ) Do I need a new test regardless?